.travis.yml

```diff
@@ -0,0 +1,23 @@
+services:
+- docker
+env:
+  global:
+  - secure: zgq+uKDXJJczx50LHWVPxPago6T/SDt/dZ6pIfb3SDNpuQLOs6k+R7rXLZW+uP7k03QD6u5cKGuuIKJgQX84PZze8PlYE7LgVTI7sBjDSL1Vp8kC8SSa4KBXtydtI3XCBLH4/SI1Q3FvtX1bCp1HvVzT4zgJq4vfF4dBZ8urCgPMsMmBcBy36yUDW5p1GOonTRoAer89WFlUOT22mDe9PECYqCAM95nZ/GICIHyU0YWP8e4L920KUqt2uduCu8kXIThJfnwbN3aqCHLoMdsdbsOcywUTHCzwFeODXQWopRKY6Zcpcm79xlp+ni2uCy8VAVb1TPJHXmJqV55aw7XxodexsR5ONG1a5Gq8EdbqtXAfPmXOyoxf6Ewc2+yrNY2amfffbXeuMaP87rmO71m24L++5v31bnGnb2GGRWaCWPgPMRjm0zweZ/h8mCc8zNLFykR62qvclevGw6pF6OIR6EbKTxmIurbXhIuxhkm7YwHBrZlPc+SfXPKX+nn3CvRI5VL3GYadhIROtU2qotdtAhezSrCl0S3yXyUlsXfKqFXfn0VGcYAvdOwkmnlK631PvgxyKvLWfTBzvSkU/LcXyImeQQBBF+8e6okoY/5flqtGabyBvrTB+3tDqI0V5ss7NOCfYhvv+nnxznDo3g/kuWTVuU2azQV5nZsZYFD2AME=
```

Comment:
Minor, but worth noting that your builds currently export some variables twice, since some are defined here and others in travis directly. Ideally we'd keep these all in one place for consistency & to prevent confusing behaviour (e.g. if you updated variable in travis, but then discovered this file overrode that anyway)